packages/vx-geo/src/components/Projection.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import { Group } from '@vx/group';
+import additionalProps from '../util/additionalProps';
+import { geoOrthographic, geoAlbers, geoMercator, geoPath } from 'd3-geo';
+
+const projectionMapping = {
+  orthographic: geoOrthographic(),
+  albers: geoAlbers(),
+  mercator: geoMercator()
+};
+
+/**
+ * Component for all projections.
+ */
+export default function Projection({
+  data,
+  projection = 'mercator',
+  projectionFunc,
+  clipAngle,
+  clipExtent,
+  scale,
+  translate,
+  center,
+  rotate,
+  precision,
+  fitExtent,
+  fitSize,
+  centroid,
+  className,
+  ...restProps
+}) {
+  const currProjection = projectionMapping[projection];
+
+  if (clipAngle) currProjection.clipAngle(clipAngle);
+  if (clipExtent) currProjection.clipExtent(clipExtent);
+  if (scale) currProjection.scale(scale);
+  if (translate) currProjection.translate(translate);
+  if (center) currProjection.translate(center);
+  if (rotate) currProjection.rotate(rotate);
+  if (precision) currProjection.rotate(precision);
+  if (fitExtent) currProjection.fitExtent(fitExtent);
+  if (fitSize) currProjection.fitSize(fitSize);
+
+  const path = geoPath().projection(currProjection);
+
+  return (
+    <Group className={`vx-${projection}-group`}>
+      {data.map((feature, i) =>
+        <g key={`${projection}-${i}`}>
+          <path
+            className={classnames(`vx-${projection}`, className)}
+            d={path(feature)}
+            {...additionalProps(restProps, feature)}
+          />
+          {centroid && centroid(path.centroid(feature), feature)}
+        </g>
+      )}
+      {projectionFunc && projectionFunc(currProjection)}
```

Comment:
@hshoff Might be helpful to use invert (https://github.com/d3/d3-geo#projection_invert) x and y coordinates to longitude and latitude. Not a good solution because you can set projection settings with props like scale, translate. Maybe its possible to place it with additionalProps on <Group ... />?